src/sentry/preprod/api/endpoints/organization_preprod_artifact_assemble.py

```diff
@@ -0,0 +1,126 @@
+import jsonschema
+import orjson
```

Comment:
We have a precommit against importing the `json` module because we want developers to use the `sentry.utils.json` interface.  Let me clarify with @asottile-sentry.  Should we have a pre-commit hook to prevent this import?

Reply:
`orjson` is "better" than sentry.utils.json but the migration was never finished

Reply:
Looks like `orjson` is used quite extensively in the Sentry code base.